python/cuda_parallel/tests/test_scan.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+import cupy as cp
+import numba.cuda
+import numba.types
+import numpy as np
+
+import cuda.parallel.experimental.algorithms as algorithms
+import cuda.parallel.experimental.iterators as iterators
+from cuda.parallel.experimental.struct import gpu_struct
+
+
+def exclusive_scan_host(h_input: np.ndarray, op, h_init):
+    result = h_input.copy()
```

Comment:
It would make sense to add `assert h_init.ndim == 1` here.

Reply:
Done